gitops/azure-devops/build.sh

```diff
@@ -1,8 +1,14 @@
-#!/bin/bash
-
-function copy_files() {
+function init() {
     cp -r * $HOME/
     cd $HOME
+
+    if [[ "$GIT_TYPE" == "github" ]]; then
+        git_dest_repo="https://github.com/$AKS_MANIFEST_REPO"
+        git_type=$GIT_TYPE
+    elif [[ "$GIT_TYPE" == "azure" ]]; then
```

Comment:
Is GIT_TYPE expected to be an environment variable? If so, maybe we should just add it to the env variables in documentation example [here](https://github.com/Microsoft/bedrock/blob/services/gitops/azure-devops/README.md)

Reply:
It is in the new [documentation](https://github.com/Microsoft/bedrock/blob/89827537f9ae04be049f0acd2a56415f0e0defe2/gitops/azure-devops/README.md) that is part of the PR

Reply:
Ok in that case it's a pipeline variable, not an env variable. That works. Resolving